samples/contrib/aws-samples/simple_train_pipeline/README.md

```diff
@@ -0,0 +1,60 @@
+# Simple pipeline with only train component
+
+An example pipeline with only train component.
+
+# Prerequisites 
+1. Install kubeflow on an EKS cluster in AWS. https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/
+2. Get and store data in S3 buckets. You can get sample data using this code  
+   `aws s3 cp s3://m-kfp-mnist/mnist_kmeans_example/data s3://<your_bucket_name>/mnist_kmeans_example/data`
+3. Prepare an IAM role with permissions to run SageMaker jobs and access to S3 buckets. https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html  
+   Go to you AWS account -> IAM -> Roles -> Create Role -> select sagemaker service -> Next -> Next -> Next -> Give some name -> Create Role -> Click on the role that you created -> Attach policy -> AmazonS3FullAccess -> Attach Policy -> Note down the role ARN
```

Comment:
nit: `Select SageMaker Service`

Also instead of telling them `Give some name`, lets provide them with a good default.